src/notes/FluxNotesEditor.jsx

```diff
@@ -1342,6 +1348,26 @@ class FluxNotesEditor extends React.Component {
             );
         }
     }
+
+    EditNoteTitleButton = () => { 
```

Comment:
[MG2] Code style: I would change EditNoteTitleButton to be camelcase (lowercase e) and add a semicolon after return.